src/test/java/test/thread/issue2361/ChildClassExample.java

```diff
@@ -0,0 +1,8 @@
+package test.thread.issue2361;
+
+import org.testng.annotations.Test;
+
+@Test(singleThreaded = true)
+public class ChildClassExample extends BaseTestClassExample {
```

Comment:
would a test case using `@Factory` behave any differently?

i experience problem with both the same, but maybe worth asserting both are fixed?

Reply:
Fixed